cpp/src/command_classes/UserCode.cpp

```diff
@@ -264,7 +264,8 @@ bool UserCode::HandleMsg
 				memcpy( data, &_data[3], size );
 				value->OnValueRefreshed( data, size );
 			} else {
-				value->OnValueRefreshed( 0x00, 1);
+				size = 0;
+				value->OnValueRefreshed( (uint8 const*)&size, 1);
 			}
```

Comment:
github uses branch based PRs, so you don't need to close the PR and create a new one. If I were going to make the above change, then I would do the following:
```
git checkout branch-which-contains-the-changes
edit the files
git add the-files
git commit --amend
git push -f my-remote branch-name
```
This will replace the old commit with the new one, however, no email notification will be sent out, so I normally go and add a comment to the PR to let reviewers know that a change has happened.

If you're dealing with multi-commit PRs, then I would add a new commit (i.e. don't use --amend) and use rebase -i to merge the new commit into the appropriate existing commit.